client/src/elements/schema-editor/schema-editor-geojson-point.js

```diff
@@ -17,29 +15,15 @@ export class SchemaEditorGeojsonPoint {
   @bindable
   showNotifications;
 
-  options = {
```

Comment:
this option and it's handling should be kept to still support the Q-Quiz tool where this is in use to have a combination of a Point geometry and a bbox.